miner/worker.go

```diff
@@ -944,7 +944,7 @@ func (w *worker) commitTransactions(env *environment, txs *types.TransactionsByP
 	}
 	// Notify resubmit loop to decrease resubmitting interval if current interval is larger
 	// than the user-specified one.
-	if interrupt != nil {
+	if interrupt != nil && atomic.LoadInt32(interrupt) >= commitInterruptResubmit {
```

Comment:
It's expected behavior. 

If no interruption signal is received, then we try to decrease the interval in case it's larger than user-specified.

Reply:
@rjl493456442 Roger, thanks for clarifying!